src/__tests__/Module-test.js

```diff
@@ -77,6 +76,7 @@ describe('Module', () => {
     (options) => createModule({...options, file: '/root/package.json'});
 
   beforeEach(function(done) {
+    process.platform = 'linux';
```

Comment:
jest supports the setup env file. I feel like we should add one and set the platform to linux from there by default? Seems to make sense since all the tests basically rely on this.